google/cloud/storage/blob.py

```diff
@@ -888,7 +890,7 @@ def _do_download(
             )
 
             while not download.finished:
-                download.consume_next_chunk(transport, timeout=timeout)
+                telemetry_wrapped_api_request(download.consume_next_chunk, transport, timeout=timeout, instrumented_function_name='GET /b/{}/o/{}'.format(self.bucket.name, self.name))
```

Comment:
NIT: this line may have issue with linting due to length, could this be broken down into shorter statements by chance?

Reply:
linted